### DISCUSSION TITLE
[iOS] [BUG] Cursor placement on blank lines

### DISCUSSION THREAD

**Discussion by chadleap** [community/community#20538] [Mobile]
**Problem**: In the iPad GitHub app, when 1-2 blank lines are created in a repository issue via carriage return key, they usually cannot have cursor positioned on them via touch-tap without creating multiple blank lines. This creates difficulty/annoyance when intending to create/insert blank lines to paste information or begin typing when editing an issue since blank lines don’t behave as expected, i.e. being selectable for cursor placement.

**Observations**: The app seems to create new blank lines with spaces which can be observed when removing these non-operable lines after desired changes are made on subsequently-created new lines to enable cursor placement. It is unclear why cursor placement is impossible without MANY blank lines being created first, which then need to be removed.

**Comment by elisealix22** [comment]
Hi @chadleap, thank you so much for the detailed bug report! I opened an issue with our iOS team to investigate. Please let us know if you're running into any other issues!
